### PR TITLE
Generate short summary to easily pickup for reporting

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
This generates short test summary to easily view stats without actually opening the report link.

Screen Shot 2019-07-22 at 10 43 03 AM

#### Ticket Link
none

### CODE REVIEW

File: e2e/run_tests.js [hunk 1]
```diff
@@ -1,13 +1,78 @@
...
+| Duration | ${(stats.duration / (60 * 1000)).toFixed(2)} mins |
+| Suites | ${stats.suites} |
+| Tests | ${stats.tests} |
+| :white_check_mark: Passed | ${stats.passes} |
+| :x: Failed | ${stats.failures} |
+| :fast_forward: Skipped | ${stats.skipped} |
+`;
+
+    let failedTests;
+    if (failedFullTitles && failedFullTitles.length > 0) {
```
Comment (srkgupta) [inline e2e/run_tests.js]: Can you please add some comment on what is being done in the subsequent steps.

File: e2e/run_tests.js [hunk 2]
```diff
@@ -1,13 +1,78 @@
...
+        const failed = failedFullTitles;
+        if (failed.length > MAX_FAILED_TITLES) {
+            failedTests = failed.slice(0, 4).map((f) => `- ${f}`).join('\n');
```
Comment (srkgupta) [inline e2e/run_tests.js]: Here 4 is hardcoded. It should be set to MAX_FAILED_TITLES-1

File: e2e/run_tests.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/run_tests.js]: Sure, now added.

File: e2e/run_tests.js (hunk 2 above)
Comment (saturninoabril) [reply e2e/run_tests.js]: Good catch, updated.

**Pull request merged**